src/main/kotlin/uk/gov/dluhc/emsintegrationapi/mapper/ProxyVoteMapper.kt

```diff
@@ -86,12 +86,12 @@ class ProxyVoteMapper(private val instantMapper: InstantMapper) {
             RejectedReasons(
                 englishReason = RejectedReason(
                     notes = englishRejectionNotes,
-                    reasons = englishRejectedReasonItems?.toList()?.map { it.type.value },
+                    reasons = englishRejectedReasonItems?.toList()?.mapNotNull { it.electorReason },
                     reasonList = englishRejectedReasonItems?.toList()?.map { item -> mapRejectedReasonItemFromEntity(item) }
                 ),
                 welshReason = RejectedReason(
                     notes = welshRejectionNotes,
-                    reasons = welshRejectedReasonItems?.toList()?.map { it.type.value },
```

Comment:
You switched the welsh one to use `englishRejectedReasonItems` for the postal one so assuming that should also happen here?